src/types/plugin-input.ts

```diff
@@ -11,14 +11,26 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
   ref: string;
 }
 
+const rolesWithReviewAuthority = T.Array(T.String(), { default: ["COLLABORATOR", "OWNER", "MEMBER"] });
```

Comment:
Shouldn't `ADMIN` be included here?

Reply:
Should `COLLABORATOR` be included? Probs not thinking about it